features/features.go

```diff
@@ -175,6 +175,13 @@ var (
 						enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
+	FeatureGateVSphereMultiDisk = newFeatureGate("VSphereMultiDisk").
+					reportProblemsToJiraComponent("splat").
+					contactPerson("vr4manta").
+					productScope(ocpSpecific).
+					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
Are you ready for techpreview or would you prefer just to start in dev preview for now?

Reply:
I can start in Dev if you prefer.  I have the initial logic completed for creating extra disks for masters and workers.  This is just the first step of the bigger picture, but i'll take your guidance on this.

Reply:
PR for MAO: https://github.com/openshift/machine-api-operator/pull/1290.  I'll also put up in top comment.

Reply:
Do you think this effort is likely to be long and span multiple releases?